frontend/selectors/basicAuth/index.js

```diff
@@ -0,0 +1,4 @@
+export function siteBasicAuth(state, siteId) {
```

Comment:
This is not used, but if it was, it wouldn't work because the state is not keyed by site id...

Reply:
thanks for caching this .. meant to delete this file